osmaxx/conversion/constants/coordinate_reference_system.py

```diff
@@ -5,7 +5,7 @@
 WGS_84 = 4326
 WGS_72 = 4322
 PSEUDO_MERCATOR = 3857
-NAD_83 = 4629
+NAD_83 = 4269
```

Comment:
Yeah, well, we hat it wrong on the [on the wiki](https://github.com/geometalab/osmaxx/wiki/Coordinate-Reference-Systems) from the very beginning and didn't properly check. :man_facepalming: